src/api/java/baritone/api/Settings.java

```diff
@@ -726,6 +726,18 @@
      */
     public final Setting<Integer> maxCachedWorldScanCount = new Setting<>(10);
 
+    /**
+     * This will only allow baritone to mine exposed ores, can be used to stop ore obfuscators on servers that use them.
+     */
+    public final Setting<Boolean> allowOnlyExposedOres = new Setting<>(false);
+
+    /**
+     * When allowOnlyExposedOres is enabled this is the distance around to search.
+     * <p>
+     * It is recommended to keep this value low, as it exponentially increases calculation times and also to keep the
```

Comment:
this sentence is incomplete and the amount of checked blocks does not increase exponentially but by (approximately) `1.3*r**3 + 2*r**2 + 2.6*r + 1`